membership_variable_period/tests/test_membership_variable_period.py

```diff
@@ -20,18 +20,21 @@ def setUp(self):
                 'membership_interval_unit': 'weeks',
             })
         self.partner = self.env['res.partner'].create({'name': 'Test'})
+        self.partner_account = self.partner.property_account_receivable_id.id
 
     def test_create_invoice_membership_product_days(self):
         self.product.membership_interval_qty = 20
         self.product.membership_interval_unit = 'days'
         invoice = self.env['account.invoice'].create(
             {'partner_id': self.partner.id,
              'date_invoice': '2015-07-01',
-             'account_id': self.partner.property_account_receivable.id,
-             'invoice_line': [(0, 0, {'product_id': self.product.id,
-                                      'name': 'Membership w/o prorrate'})]}
+             'account_id': self.partner_account,
```

Comment:
@pedrobaeza Hello, it could be better with it. As it is only a port from 8.0. As the tests are green, is it possible to merge it and then later improve it?